internal/cmd/commands.go

```diff
@@ -1028,5 +1029,42 @@ func initCommands(ui, serverCmdUi cli.Ui, runOpts *RunOptions) {
 				Func:    "remove-accounts",
 			}, nil
 		},
+
+		"workers create": func() (cli.Command, error) {
+			return &workerscmd.Command{
+				Command: base.NewCommand(ui),
+				Func:    "create",
+			}, nil
+		},
+		"workers create worker-led": func() (cli.Command, error) {
```

Comment:
Old news :-)

After discussing with Jim we felt like it made sense to do it this way. Otherwise you could forget to include an auth token and we end up calling an endpoint you didn't expect, creating a worker in the process. This way the user confirms what was intended.